obal/data/modules/koji_find_tags.py

```diff
@@ -0,0 +1,31 @@
+"""
+Find all defined Koji tags in inventory
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+def main():
+    """
+    Verify a tag
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            packages=dict(type='dict', required=True)
+        )
+    )
+
+    packages = module.params['packages']
+
+    tags = []
```

Comment:
I'd suggest to make this a `set()`
```suggestion
    tags = set()
```